Makefile

```diff
@@ -31,14 +31,21 @@ help:
 	@echo
 
 .PHONY: build
-build: .build/requirements.timestamp compile-catalog
+build: .build/requirements.timestamp compile-catalog c2cgeoform/static/dist/index.js
 
 .PHONY: check
 check: flake8 check_c2cgeoform_demo
 
 .PHONY: flake8
 flake8: .build/requirements-dev.timestamp
-	$(VENV_BIN)/flake8 c2cgeoform
+	$(VENV_BIN)/flake8 --exclude=node_modules c2cgeoform
+
+.build/node_modules.timestamp: c2cgeoform/static/package.json
+	cd c2cgeoform/static/ && npm install && cd -
```

Comment:
`cd -` is not needed here, each command in a makefile target execute from the makefile parent directory